### PR TITLE
Add hook support for organizations

### LINKED ISSUE #680: Organization webhooks support?
Hi,

Does it support organization webhook to listen to repository delete event?

### PR DESCRIPTION
**Pull request by AntonNguyen**
Closes #680 

I noticed there was support for hooks on the repository, but nothing for organization (seen at https://developer.github.com/v3/orgs/hooks/), so I've the following calls:

- create_hook,
- get_hook
- get_hooks
- edit_hook
- delete_hook

### CODE REVIEW

File: github/tests/Organization.py [hunk 1]
```diff
@@ -126,6 +129,15 @@ def testGetEvents(self):
     def testGetTeams(self):
         self.assertListKeyEqual(self.org.get_teams(), lambda t: t.name, ["Members", "Owners"])
 
+    def testCreateHookWithMinimalParameters(self):
+        hook = self.org.create_hook("web", {"url": "http://foobar.com"})
+        self.assertEqual(hook.id, 257967)
+
+    def testCreateHookWithAllParameters(self):
+        hook = self.org.create_hook("web", {"url": "http://foobar.com"}, ["fork"], False)
+        self.assertTrue(hook.active)  # WTF
```
Comment (sfdye) [inline github/tests/Organization.py]: What is this? 😄

**Comment by sfdye** [comment]
@AntonNguyen Thank you for your PR. LGTM. If it's not too much trouble, can you maybe also add the `delete` or `edit`? I guess you organization will use these actions as well. I will merge once this is more complete 👍

File: github/tests/Organization.py (hunk 1 above)
Comment (AntonNguyen) [reply github/tests/Organization.py]: I am not too sure 😂 The Respository.py test had something similar. I'll remove it, as it's not actually useful.

Comment (sfdye) [reply github/tests/Organization.py]: I mean the comment

**Comment by sfdye** [comment]
Fantastic! Thank you very much for your contribution @AntonNguyen

**Pull request merged**